app/javascript/src/utils/api/properties.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+
+export const getProperties = async (params) => {
+  const { data } = await axios.get(
+    `http://localhost:3000/api/v1/properties/?q=${params}`
+  )
+
+  return data
+}
+
+export const getProperty = async (propertyReference) => {
+  const { data } = await axios.get(
+    `http://localhost:3000/api/v1/properties/${propertyReference}`
+  )
+
+  return data
+}
```

Comment:
How about moving the `server url` into a constant variable and add a dotenv? something like that

```JS
const API = 'http://localhost:3000/'
```

```JS
API + `api/v1/properties/?q=${params}`
```